lib/discourse_dev/reviewable_user.rb

```diff
@@ -8,6 +8,7 @@ class ReviewableUser < Reviewable
     def populate!
       reasons = %i[must_approve_users invite_only suspect_user]
       @users
+        .select { |user| !user.approved? }
```

Comment:
```suggestion
        .reject(&:approved?)
```

Reply:
Would be nice to add a `not_approved` AR scope in core too:

https://github.com/discourse/discourse/blob/fcb64102b3909e7610cab3c64f764d104a86d7a2/app/models/user.rb#L303-L303